File: Two Sum II - Input array is sorted.c
int* twoSum(int* numbers, int numbersSize, int target, int* returnSize) {
    int *reval = (int*)malloc(sizeof(int)*2);
    *returnSize = 2;
    
    int left, right;
    left = 0; right = numbersSize-1;
    for(;;) {
        if(numbers[left] + numbers[right] > target)
            right--;
        else if(numbers[left] + numbers[right] < target)
            left++;
        else {
            reval[0] = left+1;
            reval[1] = right+1;
            return reval;
        }   
    } 
}

File: Two Sum II - Input array is sorted.js
var twoSum = function(numbers, target) {
    var result = [];
    for(var i = 0; i<numbers.length; i++)
    {
        for(var j = i+1; j<numbers.length; j++)
        {
            if((numbers[i] + numbers[j]) == target)
            {
                result.push(i+1)
                result.push(j+1)
                return result;
            }
        }
    }
};

